packages/core/src/loop/shared/inject-background-task-prompt.ts

```
@@ -0,0 +1,50 @@
+import type { LanguageModelV2Prompt } from '@ai-sdk/provider-v5';
+import type { AgentBackgroundConfig } from '../../background-tasks';
```

ESLint

[error] '../../background-tasks' imported multiple times.

(import/no-duplicates)